File: Sorting/python/07_K번째수.py
# fast campus 강의
# https://www.acmicpc.net/problem/11004
# 1

N, K = map(int, input().split(' '))

n_list = list(map(int, input().split(' ')))

n_list = sorted(n_list)

print(n_list[K - 1])

File: Sorting/c++/07_K번째수.cpp
// fast campus 강의
// https://www.acmicpc.net/problem/1300
// 1

#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int N, K;
vector<int> A;

int main()
{
    cin >> N >> K;
    for (int i = 0; i < N; i++) {
        int tmp;
        scanf("%d", &tmp);
        A.push_back(tmp);
    }
    sort(A.begin(), A.end());

    cout << A[K - 1];
}